homeassistant/components/sonarr/entity.py

```diff
@@ -1,6 +1,4 @@
 """Base Entity for Sonarr."""
-from __future__ import annotations
-
```

Comment:
You don't need to remove `from __future__ import annotation`. Usually it doesn't hurt and only provides a small performance improvement.

_Unless you're dealing runtime typing in some small corner cases. For more details if you're interested:_ https://pydantic-docs.helpmanual.io/usage/postponed_annotations/

Reply:
I already plan on cleaning this up in another PR. Was actually waiting for this to merge.